packages/frontend/src/components/UserSettings/SocialLoginsPanel/index.tsx

```diff
@@ -1,87 +1,100 @@
 
-const getIdentitiesQuery = async () =>
-    lightdashApi<OpenIdIdentitySummary[]>({
-        url: '/user/identities',
-        method: 'GET',
-        body: undefined,
-    });
-
-const renderIssuerType = (
+const isIssuerTypeAvailable = (
+    health: HealthState,
     issuerType: OpenIdIdentitySummary['issuerType'],
-): string => {
+) => {
     switch (issuerType) {
-        case 'google':
-            return 'Google';
-        case 'okta':
-            return 'Okta';
+        case OpenIdIdentityIssuerType.GOOGLE:
+            return !!health.auth.google.oauth2ClientId;
+        case OpenIdIdentityIssuerType.OKTA:
+            return health.auth.okta.enabled;
+        case OpenIdIdentityIssuerType.ONELOGIN:
+            return health.auth.oneLogin.enabled;
         default:
-            return issuerType;
+            return false;
     }
 };
 
 const SocialLoginsPanel: FC = () => {
-    const { data } = useQuery<OpenIdIdentitySummary[], ApiError>({
-        queryKey: 'user_identities',
-        queryFn: getIdentitiesQuery,
-    });
+    const { data: health } = useHealth();
+    const { data: userSocialLogins } = useOpenIdentities();
+
     const deleteMutation = useDeleteOpenIdentityMutation();
+
     return (
-        <div>
-            <CardWrapper>
-                {data?.map((id) => (
-                    <CardContainer key={id.issuerType} elevation={0}>
-                        <Text>
-                            <Bold className={Classes.TEXT_OVERFLOW_ELLIPSIS}>
-                                {renderIssuerType(id.issuerType)}
-                            </Bold>
-                            {id.email && <Tag minimal>{id.email}</Tag>}
-                        </Text>
-                        <Button
-                            icon="delete"
-                            disabled={deleteMutation.isLoading}
-                            intent="danger"
-                            text="Delete"
-                            outlined
-                            onClick={() =>
-                                deleteMutation.mutate({
-                                    email: id.email,
-                                    issuer: id.issuer,
-                                })
-                            }
-                        />
-                    </CardContainer>
-                ))}
-            </CardWrapper>
-            <Title>Add social login</Title>
-            <GoogleButtonWrapper>
-                <GoogleLoginButton />
-            </GoogleButtonWrapper>
-            <GoogleButtonWrapper>
-                <OktaLoginButton />
-            </GoogleButtonWrapper>
-            <GoogleButtonWrapper>
-                <OneLoginLoginButton />
-            </GoogleButtonWrapper>
-        </div>
+        <Stack spacing="md">
+            {health &&
+                (
+                    Object.entries(userSocialLogins ?? {}) as Entries<
+                        typeof userSocialLogins
```

Comment:
We could probably do the following in a `entries.d.ts`

```ts
declare global {   
  interface ObjectConstructor {    
     entries<T extends object>(o: T): Entries<T>  
    }
 }
```

but don't know how others  feel with overriding this stuff 😅